src/app/components/form/DualCurrencyField/index.tsx

```diff
@@ -4,7 +4,7 @@ import { classNames } from "~/app/utils";
 export type Props = {
   suffix?: string;
   endAdornment?: React.ReactNode;
-  fiatValue: string | number;
+  fiatValue: string;
```

Comment:
that means we will always have a formatted string in there? 

handling amounts is something I think we need to be careful to not be confusing: 
where do we use strings, numbers, bignumbers, ...? 
is it formatted? how to we calculate with it?

Reply:
This is a valid correction I believe. They value is initaly being created by `getFiatValue` which always returns a string anyway.

Reply:
Yes exactly. Every fiatValue is [formatted here](https://github.com/getAlby/lightning-browser-extension/blob/665ccd0cf611e31f82b07a9a2953707d22bd9db9/src/common/utils/currencyConvert.ts#L96-L111) which always returns a string.